pubsub/pubsub_test.go

```diff
@@ -184,18 +183,21 @@ func benchmarkNClients(n int, b *testing.B) {
 			for range ch {
 			}
 		}()
-		s.Subscribe(ctx, clientID, query.MustParse(fmt.Sprintf("abci.Account.Owner = 'Ivan' AND abci.Invoices.Number = %d", i)), ch)
+		s.Subscribe(ctx, clientID, query.
+			MustParse(
+				fmt.Sprintf("abci.Account.Owner = 'Ivan' AND abci.Invoices.Number = %d", i)), ch)
 	}
 
 	b.ReportAllocs()
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		s.PublishWithTags(ctx, "Gamora", map[string]interface{}{"abci.Account.Owner": "Ivan", "abci.Invoices.Number": i})
+		s.PublishWithTags(ctx, "Gamora",
+			map[string]interface{}{"abci.Account.Owner": "Ivan", "abci.Invoices.Number": i})
 	}
 }
 
 func benchmarkNClientsOneQuery(n int, b *testing.B) {
-	s := pubsub.NewServer()
+	s := pubsub.NewServer(nil)
```

Comment:
and I don't like this... when some dev looks at this line, how should he guess what nil is??? solution could be using options (like `pubsub.NewServer()` & `pubsub.NewServer(Logger(log.TestingLogger()))`). But I don't see how current `SetLogger()` is different.

Reply:
or we can pass `log.NopLogger()`, but not nil, please